source/compiler/sc1.c

```diff
@@ -3798,7 +3798,7 @@ static int newfunc(char *firstname,int firsttag,int fpublic,int fstatic,int stoc
       ptr=ptr->next;
     } /* while */
   } /* if */
-  startfunc(sym->name); /* creates stack frame */
+  startfunc(sym->name, sym->flags & flagNAKED); /* creates stack frame */
```

Comment:
The styling followed by the project requires that you do not add a space between arguments.